dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/EditWebHook.cshtml

```diff
@@ -39,5 +41,5 @@
         <govuk-checkboxes-item value="@true">Regenerate secret</govuk-checkboxes-item>
     </govuk-checkboxes>
 
-    <govuk-button type="submit">Save</govuk-button>
+    <govuk-button type="submit" asp-page-handler="Save" value="save">Save</govuk-button>
```

Comment:
Can we keep this as the regular page handler method i.e. no `asp-page-handler`?

Reply:
Done